README.md

```diff
@@ -626,13 +627,13 @@ If you or your company would like to support this project, please consider:
 
 If you would like to support us in some other way, please contact with creating an issue.
 
-## Sponsors
+## <a id="sponsors"></a>Sponsors
 
-### [JetBrains](https://www.jetbrains.com/)
+### <a id="JetBrains"></a>[JetBrains](https://www.jetbrains.com/)
 
 They are providing a free license of their IDEs for this OSS development.   
 
-### [1Password](https://github.com/1Password/1password-teams-open-source)
+### <a id="1password"></a>[1Password](https://github.com/1Password/1password-teams-open-source)
```

Comment:
Why did you add a tag?

Reply:
The tag makes a direct reference to the sponsor when clicked on at the sponsors from the list items towards the top in Readme.

Reply:
Is it needed? I think header TOC items already links to each Markdown header.

Reply:
Actually 1Password was not present on the list. So while adding it, I modified it below as well by adding a tag. It wasn't necessary but added to make the complete thing.

Only its inclusion in the sponsors section was necessary.

Reply:
I removed the tag now. So only its inclusion in the list was the change. :smile:

Reply:
I added the references to development section and also styled the single line codes in "How to use" section.